GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.Designer.cs

```diff
@@ -164,74 +204,92 @@ private void InitializeComponent()
-            this.contextMenuStripRepository.Closed += new System.Windows.Forms.ToolStripDropDownClosedEventHandler(this.contextMenuStrip_Closed);
-            this.contextMenuStripRepository.Opening += new System.ComponentModel.CancelEventHandler(this.contextMenuStrip_Opening);
-            // 
-            // contextMenuStripCategory
-            // 
-            this.contextMenuStripCategory.Items.AddRange(new System.Windows.Forms.ToolStripItem[] {
-                this.tsmiCategoryRename,
-                this.tsmiCategoryDelete,
-                this.tsmiCategoryClear});
-            this.contextMenuStripCategory.Name = "contextMenuStripCategory";
-            this.contextMenuStripRepository.Size = new System.Drawing.Size(225, 66);
-            // 
             // tsmiRemoveFromList
             // 
             this.tsmiRemoveFromList.Name = "tsmiRemoveFromList";
-            this.tsmiRemoveFromList.Size = new System.Drawing.Size(224, 22);
+            this.tsmiRemoveFromList.Size = new System.Drawing.Size(273, 22);
             this.tsmiRemoveFromList.Text = "Remove project from the list";
             this.tsmiRemoveFromList.Click += new System.EventHandler(this.tsmiRemoveFromList_Click);
             // 
             // tsmiRemoveMissingReposFromList
             // 
             this.tsmiRemoveMissingReposFromList.Name = "tsmiRemoveMissingReposFromList";
-            this.tsmiRemoveMissingReposFromList.Size = new System.Drawing.Size(224, 22);
+            this.tsmiRemoveMissingReposFromList.Size = new System.Drawing.Size(273, 22);
             this.tsmiRemoveMissingReposFromList.Text = "Remove missing projects from the list";
             this.tsmiRemoveMissingReposFromList.Click += new System.EventHandler(this.tsmiRemoveMissingReposFromList_Click);
             // 
-            // tsmiOpenFolder
+            // imageList1
             // 
-            this.tsmiOpenFolder.Image = global::GitUI.Properties.Images.BrowseFileExplorer;
-            this.tsmiOpenFolder.Name = "tsmiOpenFolder";
-            this.tsmiOpenFolder.Size = new System.Drawing.Size(224, 22);
-            this.tsmiOpenFolder.Text = "Show in folder";
-            this.tsmiOpenFolder.Click += new System.EventHandler(this.tsmiOpenFolder_Click);
+            this.imageList1.ImageStream = ((System.Windows.Forms.ImageListStreamer)(resources.GetObject("imageList1.ImageStream")));
+            this.imageList1.TransparentColor = System.Drawing.Color.Transparent;
+            this.imageList1.Images.SetKeyName(0, "source_code.png");
+            // 
+            // textBoxSearch
+            // 
+            this.textBoxSearch.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.textBoxSearch.Location = new System.Drawing.Point(3, 3);
+            this.textBoxSearch.Name = "textBoxSearch";
+            this.textBoxSearch.Size = new System.Drawing.Size(439, 20);
+            this.textBoxSearch.TabIndex = 0;
+            this.textBoxSearch.PlaceholderText = "Search";
```

Comment:
I think this string should be set in the constructor and using the `TranslationString` so that the string could be translated.

Also, I don't know what other think but is "Search a repository" clearer?

Reply:
"Search repositories" seems more idiomatic to me.